tests/unit_tests/test_routes.py

```diff
@@ -75,3 +75,43 @@ def test_get_id_nonexistent(self):
         self.assertEqual(
             response.json['error']['message'], f'Could not find {id_to_fetch} within the database.'
         )
+
+
+class TestPutEndpoint(BaseRouteTestCase):
+
+    def test_valid_kml_put(self):
+
+        response = self.app.post(
+            "/kml", data=self.kml_string, content_type="application/vnd.google-earth.kml+xml"
+        )
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(response.content_type, "application/json")
+
+        id_to_fetch = response.json['id']
```

Comment:
just for clarity, perhaps use `id_to_put` or `id_to_update` here, instead of `id_to_fetch`